### DISCUSSION TITLE
Can I use Endless Knobs? how?

### DISCUSSION THREAD

**Discussion by andperoty** [rsjaffe/MIDI2LR#649] [Q&A] [answered]
My  Behringer CMD DC-1 have 12 endless knobs.
But all them only show up in the MIDI2LR as [63] and [65]  (left and rigth)

I find out some people use Bome midi translator as a workout.

if pp>=64 then ga=ga+1
if pp<=64 then ga=ga-1
if ga<=0 then ga=0
if ga>=127 then ga=127

But that could be a native option on MIDI2LR

![image](https://user-images.githubusercontent.com/20983757/163890616-459a8482-cb52-4535-9751-4c0d9e933cca.png)

**Comment by rsjaffe** [comment, marked as answer]
Yes. See the wiki https://github.com/rsjaffe/MIDI2LR/wiki/MIDI-Controller-Setup#controllers-that-send-relative-values-or-have-unusual-ranges

**Reply by andperoty** [reply]
Ty so much!
